website/docs/cloud-docs/workspaces/settings/run-tasks.mdx

```diff
@@ -6,22 +6,26 @@ description: >-
+Terraform Cloud run tasks let you directly integrate third-party tools and services at certain stages in the Terraform Cloud run lifecycle. You can use run tasks to validate Terraform configuration files, analyze plan outputs before applying them, scan for security vulnerabilities, or perform other custom actions.
 
-You can manage run tasks through the UI or through the [Run Tasks APIs](/terraform/cloud-docs/api-docs/run-tasks/run-tasks).
+Each run task in a workspace can send information to an external service at [specific run stages](#understanding-run-tasks-within-a-run). The external service processes the information and lets Terraform Cloud know if that task passed or failed. Terraform Cloud then uses this information to determine if a run should proceed.
+
+You can manage run tasks through the Terraform Cloud UI or the [Run Tasks API](/terraform/cloud-docs/api-docs/run-tasks/run-tasks).
 
 > **Hands-on:** Try the [HCP Packer validation run task](/packer/tutorials/hcp/setup-tfc-run-task) tutorial.
 
 ## Requirements
 
-**Terraform Version** - You can only assign run tasks to workspaces that use a Terraform version of 1.1.9 and later. You can downgrade a workspace with existing runs to use a prior Terraform version without causing an error. However, Terraform Cloud no longer triggers the run tasks during plan and apply operations.
+**Terraform Version** - You can assign run tasks to workspaces that use a Terraform version of 1.1.9 and later. You can downgrade a workspace with existing runs to use a prior Terraform version without causing an error. However, Terraform Cloud no longer triggers the run tasks during plan and apply operations.
 
 **Permissions** - To create a run task, you must have a user account with the [Manage Run Tasks permission](/terraform/cloud-docs/users-teams-organizations/permissions#manage-run-tasks). To associate run tasks with a workspace, you need the [Manage Workspace Run Tasks permission](/terraform/cloud-docs/users-teams-organizations/permissions#general-workspace-permissions) on that particular workspace.
 
 ## Creating a Run Task
 
+When a run task is triggered, it sends an API payload to the specied external service. This payload contains a collection of run-related information, including a callback URL which the service uses to return a pass or fail status to Terraform Cloud. Terraform Cloud uses this status response to determine if a run should proceed, based on the run task's enforcement settings.
```

Comment:
we have something very similar a few paragraphs up, so i want to keep this one very brief and specific to implementation. the last sentence still feels a little repetitive.. do you think we need it? 
```suggestion
Run tasks send an API payload an external service. This payload contains run-related information, including a callback URL which the service uses to return a pass or fail status to Terraform Cloud. Terraform Cloud uses this status response to determine if a run should proceed, based on the run task's enforcement settings.
```